examples/python/shift_scheduling_sat.py

```diff
@@ -215,11 +215,11 @@ def solve_shift_scheduling(params, output_proto):
     # Request: (employee, shift, day, weight)
     # A negative weight indicates that the employee desire this assignment.
     requests = [
-        # Employee 3 wants the first Saturday off.
+        # Employee 3 wants the first shift of Saturday off.
```

Comment:
I would rephrase it too to be consistency:
"Employee 3 does not want the first shift on the first Saturday (negative weight)."

Reply:
and please against master

Reply:
@lperron The pull request goes into google:master. Does this fulfill your request?

Reply:
I guess laurent was trick by the name of your branch stable otherwise everything is good !
thanks for the PR